src/components/AssetInfo/TokenSideBarField.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import css from "./TokenSideBar.scss";
+
+interface TokenSideBarFieldProps {
+  id: string;
+  title: string;
+  ctaText: string;
+  ctaStatus?: string;
+  children?: React.ReactNode;
+  handleClick: (e: any) => void;
+}
+
+const TokenSideBarField = ({ id, title, ctaText, ctaStatus, children, handleClick }: TokenSideBarFieldProps) => {
```

Comment:
assuming CTA means call-to-action, maybe we should just name these fields textField and statusField? not too sure why CTA is relevant

Reply:
ok, updated the text.